analysis/topEFT/topeft.py

```diff
@@ -318,12 +326,8 @@ def process(self, events):
             pMC     = ak.prod(bJetEff       [isBtagJetsMedium], axis=-1) * ak.prod((1-bJetEff       [isNotBtagJetsMedium]), axis=-1)
             pData   = ak.prod(bJetEff_data  [isBtagJetsMedium], axis=-1) * ak.prod((1-bJetEff_data  [isNotBtagJetsMedium]), axis=-1)
             pDataUp = ak.prod(bJetEff_dataUp[isBtagJetsMedium], axis=-1) * ak.prod((1-bJetEff_dataUp[isNotBtagJetsMedium]), axis=-1)
-            pDataDo = ak.prod(bJetEff_dataDo[isBtagJetsMedium], axis=-1) * ak.prod((1-bJetEff_dataDo[isNotBtagJetsMedium]), axis=-1)
-
+            pDataDo = ak.prod(bJetEff_dataDo[isBtagJetsMedium], axis=-1) * ak.prod((1-bJetEff_dataDo[isNotBtagJetsMedium]), axis=-1)           
             pMC      = ak.where(pMC==0,1,pMC) # removeing zeroes from denominator...
-            btagSF   = pData  /pMC
-            btagSFUp = pDataUp/pMC
-            btagSFDo = pDataDo/pMC
```

Comment:
Were these lines removed on accident?

Reply:
@bryates, I think these lines were removed because they are no longer necessary (though @andrea21z should of curse correct me if I'm misremembering). Andrea is now passing e.g. `pDataUP/pMC` to the `add()` method [here](https://github.com/TopEFT/topcoffea/blob/adding_MuonScaleCor/analysis/topEFT/topeft.py#L346), instead of calculating this variable beforehand. This was to work around a "feature" of `class coffea.processor.Weights`, where the input values can be modified in place by `add()` (documented [here](https://coffeateam.github.io/coffea/api/coffea.processor.Weights.html?highlight=weight#coffea.processor.Weights.add)).

Reply:
No, I removed them because we are introducing these calculations directly in the `weights_dict.add` to fix the problem we found some weeks ago.

Reply:
Ok, thanks. I guess I didn't follow the exact code that went into that fix.